tests/Cache/CacheClientTest.php

```diff
@@ -372,4 +372,72 @@ public function testDelete() {
         $this->assertNotNull($response->asMiss());
     }
 
+    // List API tests
+
+    public function testListPushFrontFetchHappyPath() {
+        $listName = uniqid();
+        $value = uniqid();
+        $response = $this->client->listPushFront($this->TEST_CACHE_NAME, $listName, $value, true, 6000);
+        $this->assertNotNull($response->asSuccess());
+        $response = $this->client->listFetch($this->TEST_CACHE_NAME, $listName);
+        $this->assertNotNull($response->asHit());
+        $values = $response->asHit()->values();
+        $this->assertNotEmpty($values);
+        $this->assertCount(1, $values);
+        $this->assertContains($value, $values);
+        $response = $this->client->listPushFront($this->TEST_CACHE_NAME, $listName, 'value', true, 6000);
+        $this->assertNotNull($response->asSuccess());
+        $response = $this->client->listFetch($this->TEST_CACHE_NAME, $listName);
+        $this->assertNotNull($response->asHit());
+        $values = $response->asHit()->values();
+        $this->assertNotEmpty($values);
+        $this->assertCount(2, $values);
```

Comment:
nit - worth adding one more assertion to ensure the expected values in the expected order?

Reply:
Sure, added in 63c78c5.

Reply:
fantastic, thanks